hot-reload-orchestration/src/test/kotlin/org/jetbrains/compose/reload/orchestration/benchmarks/ServerClientBenchmark.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024-2025 JetBrains s.r.o. and Compose Hot Reload contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+@file:Suppress("unused")
+
+package org.jetbrains.compose.reload.orchestration.benchmarks
+
+import kotlinx.benchmark.Benchmark
+import kotlinx.benchmark.BenchmarkMode
+import kotlinx.benchmark.Measurement
+import kotlinx.benchmark.Mode
+import kotlinx.benchmark.Scope
+import kotlinx.benchmark.Setup
+import kotlinx.benchmark.State
+import kotlinx.benchmark.TearDown
+import kotlinx.benchmark.Warmup
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.compose.reload.core.awaitOrThrow
+import org.jetbrains.compose.reload.core.getOrThrow
+import org.jetbrains.compose.reload.orchestration.OrchestrationClient
+import org.jetbrains.compose.reload.orchestration.OrchestrationClientRole
+import org.jetbrains.compose.reload.orchestration.OrchestrationMessage
+import org.jetbrains.compose.reload.orchestration.OrchestrationServer
+import org.jetbrains.compose.reload.orchestration.TestOrchestrationState
+import org.jetbrains.compose.reload.orchestration.connectOrchestrationClient
+import org.jetbrains.compose.reload.orchestration.startOrchestrationServer
+import org.jetbrains.compose.reload.orchestration.stateKey
+import java.util.concurrent.TimeUnit
+
+@State(Scope.Benchmark)
+open class ServerClientBenchmark {
```

Comment:
Checked in because I was curious if something in the code might make things slow